src/scicookie/hooks/post_gen_project.py

```diff
@@ -258,14 +257,6 @@ def http2ssh(url):
     return url.replace("/", ":", 1)
 
 
-def validation():
```

Comment:
Please check if this function is being called anywhere and remove all those occurences